Update README.md (#7068)
microsoft/semantic-kernel: README.md

### For C#:

1. Go to the Quick start page [here](https://learn.microsoft.com/en-us/semantic-kernel/get-started/quick-start-guide?pivots=programming-language-csharp) and follow the steps to dive in.
2. After Installing the SDK, we advise you follow the steps and code detailed to write your first console app.
   ![dotnetmap](https://learn.microsoft.com/en-us/semantic-kernel/media/dotnetmap.png)

### For Python:

1. Go to the Quick start page [here](https://learn.microsoft.com/en-us/semantic-kernel/get-started/quick-start-guide?pivots=programming-language-csharp) and follow the steps to dive in.

> 2. You'll need to ensure that you toggle to C# in the the Choose a programming lanugage table at the top of the page.

Reviewer: ```suggestion
2. You'll need to ensure that you toggle to C# in the the Choose a programming language table at the top of the page.
```

Once you've finished the getting started notebooks, you can then check out the main walkthroughs
on our Learn site. Each sample comes with a completed C# and Python project that you can run locally.

> 1. 📖 [Getting Started]((https://learn.microsoft.com/en-us/semantic-kernel/get-started/quick-start-guide)

Reviewer: ```suggestion
1. 📖 [Getting Started](https://learn.microsoft.com/en-us/semantic-kernel/get-started/quick-start-guide)
```

- Read the [documentation](https://aka.ms/sk/learn)
- Learn how to [contribute](https://learn.microsoft.com/en-us/semantic-kernel/get-started/contributing) to the project

> - Ask questions in the [GitHub Discussion community](https://aka.ms/SKDiscord)

Reviewer: Should this be Discord?

```suggestion
- Ask questions in the [GitHub discussions](https://github.com/microsoft/semantic-kernel/discussions)
- Ask questions in the [Discord community](https://aka.ms/SKDiscord)

```
